alliance/config/local_settings.py

```diff
@@ -0,0 +1,12 @@
+DATABASES = {
+	'default': {
+		'ENGINE': 'django.db.backends.postgresql_psycopg2',
+		'NAME': 'northbr6_devwaterwheel',
+		'USER': 'alliance',
+		'PASSWORD': 'beloved',
+		'HOST': '127.0.0.1',
+		'PORT': '5432',
+    }
+}
+
+SECRET_KEY = '-ccj-m$@5h9z$t%+9zq6z$y@s%e9+kapdy^ozt4k^lfvpq)bxm'
```

Comment:
So if the env variable is set to some value, the runtime ignores local_settings, right?

OK.